assignment.js

```diff
@@ -65,6 +73,9 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
+let values = [57,13,4,38]
+
+console.log(Math.max.apply(null, values))
```

Comment:
@Defy1x nice use of `apply()` - it's an advanced approach, just make sure you understand how it works